udev/set-paths.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/meson.build b/meson.build
+index 3593eac710..2582dcab8c 100644
+--- a/meson.build
++++ b/meson.build
+@@ -6,8 +6,8 @@ project('systemd', 'c',
+         default_options: [
+                 'c_std=gnu99',
+                 'prefix=/usr',
+-                'sysconfdir=/etc',
+-                'localstatedir=/var',
++                'sysconfdir=/app/etc',
++                'localstatedir=/app/var',
+         ],
+         meson_version : '>= 0.46',
+        )
+@@ -70,7 +70,7 @@ endif
+ conf.set10('HAVE_SPLIT_BIN', split_bin,
+            description : 'bin and sbin directories are separate')
+ 
+-rootprefixdir = get_option('rootprefix')
++rootprefixdir = '/app'
```

Comment:
Why are you patching this instead of just `-Drootprefix=/app`?

Reply:
I actually tried via `config-opts` to pass those, but that didn't seem to work out. But I'll test it again to be sure, not that I just missed something or did a typo or so...

Reply:
Okay it works with `config-opts`... phew... And I know why it didn't work before: it's supposed to be an array, not a single string... flatpak just gives a warning at top, which I evidently missed...